state/user.go

```diff
@@ -186,7 +191,6 @@ func (st *State) updateExistingUser(u *User, displayName, password, creator stri
 		Assert: txn.DocExists,
 		Update: updateUser,
 	}}
-	updateUserOps = append(updateUserOps, removeControllerOps...)
```

Comment:
These really need to done in the same transaction. The problem here is that there is no get to check the the controller even has this user before adding the ops from removeControllerUserOps.